combat-harden.lua

```diff
@@ -0,0 +1,131 @@
+-- Sets a unit's combat-hardened value to a given percent
+--@ module = true
+
+local help = [====[
+
+combat-harden
+=============
+Sets the combat-hardened value on a unit, making them care more/less about seeing corpses.
+Requires a value and a target.
+
+Valid values:
+
+:value <0-100>:
```

Comment:
My thoughts (for when I'm back online and can merge this, but feel free to change this too if you're monitoring this PR): it might not be clear that you need the `-` in front of the arguments here (`-tier` instead of `tier`, `-all`, etc.), so I would specify that. A bunch of the modtools use a different style for this ([example](https://github.com/DFHack/scripts/blob/master/modtools/create-item.lua)) although I don't really like how that displays in the [HTML docs](https://dfhack.readthedocs.io/en/stable/docs/_auto/modtools.html#modtools-create-item). Adding the `-`s is probably enough, honestly, and I can take care of that when I'm able to merge this.